client/question/services/FeedbackGeneratorService.js

```diff
@@ -53,6 +61,57 @@ tie.factory('FeedbackGeneratorService', [
     var consecutiveSameRuntimeErrorCounter = 0;
 
     /**
+     * Object containing variations on feedback text for different scenarios.
+     *
+     * $type {Object.<string, Array.<string>>}
```

Comment:
Follow existing docstring conventions please -- `@type` not `$type`. Ditto below.

Reply:
Ah, that was typo.